example/simple_example.py

```diff
@@ -23,27 +28,27 @@ def is_error200(code):
 # buckets is the internavals for histogram parameter. buckets is a optional parameter
 # error_fn is a function to define what http status code is a error. By default errors are
 # 400 and 500 status code. error_fn is a option parameter
-register_metrics(app, buckets=[0.3, 0.6], error_fn=is_error200)
+register_metrics(app, buckets=[0.3, 0.6], error_fn=is_error200, registry=registry)
 
 # Plug metrics WSGI app to your main app with dispatcher
-dispatcher = DispatcherMiddleware(app.wsgi_app, {"/metrics": make_wsgi_app()})
+dispatcher = DispatcherMiddleware(app.wsgi_app, {"/metrics": make_wsgi_app(registry=registry)})
 
 # a dependency healthcheck
 def check_db():
     try:
         response = req.get("http://localhost:5000/database")
-        if response.status_code == 200:
-            return 1
+        app.logger.info(response)
+        return 1
```

Comment:
@Ziul, an exception would be thrown if `response` have status code 400 or 500? If not, I think we should consider it on the value we return.

Reply:
A timeout or invalid DNS would throw an exception. But the callback expects a boolean or 0/1 value so I updated the code using `response.status_code < 400` as the return inside the `try`.